api-docs/v2.2/ref.yml

```diff
@@ -13389,15 +14277,18 @@ paths:
         '204':
           description: >-
             InfluxDB validated the request data format and accepted the data for
-            writing to the bucket. `204` doesn't indicate a successful write
-            operation since writes are asynchronous. See [how to check for write
+            writing to the bucket. Because data is written to InfluxDB
```

Comment:
Should there be a link here?   I see one bracket in the middle fo the sentence.

See [how to check for write
            writing to the bucket.